doc/requirements.txt

```diff
@@ -1,2 +1,2 @@
 Sphinx
-sphinx-book-theme
\ No newline at end of file
+sphinx-book-theme
```

Comment:
this looks the same to me...

Reply:
likewise 😕. must be an extra space or something?

Reply:
ah, it adds a new line to the end of the file